<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeGroupForm</class>
 <widget class="QDialog" name="EmployeeGroupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 254, 178);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>10</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Поиск по огр. полномочий</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>39</y>
     <width>369</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="lbAuthorityLimitation">
      <property name="text">
       <string>Ограничение полномочий</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:green;
font-size: 36px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Группа сотрудников&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbNumber">
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:green;
font-size: 36px</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Добавление группы сотрудников&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditGroupNumber">
      <property name="styleSheet">
       <string notr="true">border: 1px solid black; background-color:white;
</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBoxAuthorityLimitation">
      <property name="styleSheet">
       <string notr="true">background-color:white; border: 1px solid black; border-radius: 0px; padding: 1px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>371</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="2">
     <widget class="QPushButton" name="delete_2">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="save">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="add">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="previous">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="logout">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="next">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Вперед</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="cancel">
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: lightgreen</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="employeegroup">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightgreen</string>
   </property>
   <property name="text">
    <string>Открыть отчёт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
